### DISCUSSION TITLE
API ?

### DISCUSSION THREAD

**Discussion by kiarn** [JeromeDevome/GRR#387] [Q&A]
**Votre demande de fonctionnalité est-elle liée à un problème ? Décrivez s'il vous plait.**

Ce n'est pas lié à un problème de GRR, mais plutôt à la recherche de la simplification de notre organisation.
Nous utilisons GRR dans une école dans laquelle nous avons 3 différents calendriers à gérer (emplois du temps, calendrier annuel et réservations). Le blocage d'une journée banalisée (sortie ou autre) demande de modifier les 3 calendriers. Nous cherchons donc une solution pour synchroniser les 3 calendriers. 

**Décrivez la solution que vous souhaitez**

Simple question: existe-t-il une API permettant de gérer les réservations de GRR ?
Si non, est-il prévu d'en implémenter une ?

Merci !

**Comment by JeromeDevome** [comment]
Bonjour pas à ce jour, dans un avenir plus ou moins loin, oui nous espérons avoir une api.
